docs/content/comparisons/druid-vs-storage-formats.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: doc_page
+---
+
+Druid vs Storage Formats (Parquet/Kudu)
```

Comment:
Parquet may make more sense under "SQL on Hadoop" than here, as it is often used by those SQL engines to speed up queries.

So if we did that, this page would become just about Kudu.